src/WarcraftLegacies.Source/Quests/Scourge/QuestEnKilahUnlock.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using MacroTools.ControlPointSystem;
+using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.FactionBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.ObjectiveSystem.Objectives.TimeBased;
+using MacroTools.QuestSystem;
+using WarcraftLegacies.Source.Setup.Legends;
+using WCSharp.Shared.Data;
+using static War3Api.Common;
+
+namespace WarcraftLegacies.Source.Quests.Scourge
+{
+  /// <summary>
+  /// Capture EnKilah's control point to gain control of all buildings and units in the area.
+  /// </summary>
+  public sealed class QuestEnKilahUnlock : QuestData
+  {
+    private readonly List<unit> _rescueUnits;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestEnKilahUnlock"/> class.
+    /// </summary>
+    public QuestEnKilahUnlock(Rectangle rescueRect) : base(
+      "Temple City of En'kilah", "The Temple City of En'kilah will be the perfect place for an outpost near the Borean Tundra.",
```

Comment:
```suggestion
      "Temple City of En'kilah", "The temple city of En'kilah will be the perfect place for an outpost near the Borean Tundra.",
```
Neither temple nor city are proper nouns so no need for the capital. Can you fix this in the rest of the quest also please?